src/wizard/unity/index.md

```diff
@@ -9,6 +9,8 @@ type: framework
 
 Get the SDK via the [Unity Package Manager using a Git URL](https://docs.unity3d.com/Manual/upm-ui-giturl.html) to Sentry's SDK repository:
 
+> Make sure not empty space is appended to the end of the URL, otherwise Unity fails to find the package.
```

Comment:
```suggestion
> Make sure trimming empty spaces appended to the end of the URL, otherwise, The Unity Package Manager fails to find the package.
```

not sure if any better, @PeloWriter